src/components/clusterWizard/ClusterDetails.tsx

```diff
@@ -147,7 +147,7 @@ const ClusterDetailsForm: React.FC<ClusterDetailsFormProps> = (props) => {
 
     try {
       const { data } = await postCluster(params);
-      history.push(`${routeBasePath}/clusters/${data.id}`);
+      history.push(`${routeBasePath}/clusters/${data.id}?flow=new`);
```

Comment:
to avoid exposing the logic to the user we could use `location.state`
```suggestion
      history.push(`${routeBasePath}/clusters/${data.id}`, { wizardFlow: 'new' });
```